### PR TITLE
Add user with variable

### LINKED ISSUE #2501: [FR] Non-interactive, environment variable based set-up in Docker deployments
**Is your feature request the result of a bug?**
No. It is a quality of life improvement.

**Problem**
As discussed on the side in #2454, Docker-based deployment could be simplified a little bit. Some systems, in particular Synology NAS, do not have docker-compose nor do they allow for easy terminal access.

**Suggested solution**
I could also see an approach where bootstrapping is done through environment variables which could be removed by the user once it's set up (and then heavily complains about it if the user doesn't unset them afterwards), e.g., `INVENTREE_ADMIN_USER` and `INVENTREE_ADMIN_PASSWORD`. A start-up script could check for the presence of these, run migrations and create user accounts accordingly. That would probably be a more generic approach that might also work for other non-Synology deployment situations.

**Describe alternatives you've considered**
None on a generic level; for Synology NAS systems, a Synology package could work.

**Examples of other systems**
Show how other software handles your FR if you have examples.

**Do you want to develop this?**
Please no, my bash is terrible 😅

### PR DESCRIPTION
**Pull request by matmair**
This PR adds the possibility to add admin users with environment variables or via the config.

Closes #2501

<a href="https://gitpod.io/#https://github.com/inventree/InvenTree/pull/2689"></a>

### CODE REVIEW

File: InvenTree/InvenTree/apps.py [hunk 1]
```diff
@@ -138,3 +146,54 @@ def update_exchange_rates(self):
...
+        if set_variables < 3:
+            logger.warn('Not all required settings for adding a user on startup are present:\nINVENTREE_SET_USER, INVENTREE_SET_EMAIL, INVENTREE_SET_PASSWORD')
+            settings.USER_ADDED = True
+            return
+
+        # good to go -> create user
+        user = get_user_model()
+        try:
+            with transaction.atomic():
+                new_user = user.objects.create_user(add_user, add_email, add_password)
```
Comment (SchrodingersGat) [inline InvenTree/InvenTree/apps.py:191]: The new user should be assigned superuser status, right?

File: InvenTree/config_template.yaml [hunk 2]
```diff
@@ -154,6 +154,11 @@ static_root: '/home/inventree/data/static'
 # Use environment variable INVENTREE_LOGIN_ATTEMPTS
 #login_attempts: 5
 
+# Add new user on first startup
```
Comment (SchrodingersGat) [inline InvenTree/config_template.yaml:157]: Should these be called `admin_user` / `admin_email` / `admin_password` for improved clarity?

**Comment by matmair** [comment]
@schrodingersGat added the changes you asked for!

**Comment by matmair** [comment]
Replaced with #2706

**Pull request closed without merging**